src/core/MouseCapture.tsx

```diff
@@ -150,12 +150,15 @@ export default class MouseCapture extends React.PureComponent<Props, State> {
   };
 
   private _onWheel = (e: React.WheelEvent<HTMLElement>) => {
-    if (this.props.onZoom && e.deltaY) {
+    // In Chrome, shift + wheel results in horizontal scrolling and
+    // deltaY == 0 while deltaX != 0, and deltaX should be used instead
+    const delta = e.deltaY || e.deltaX;
```

Comment:
In order to avoid erroneously capturing horizontal scroll events, can you narrow the conditional to only do this when shift is being used?